src/doc/yaml/cms.yaml

```diff
@@ -0,0 +1,565 @@
+        - Credential
+      summary: (Un)suspend a credential
+      operationId: suspendCredential, unsuspendCredential
+      parameters:
+        - name: credentialId
+          in: path
+          description: the id of the credential
+          required: true
+          schema:
+            type: string
+        - name: transactionId
+          in: query
+          description: The id of the transaction
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/SuspendCredential'
+      responses:
+        '204':
+          description: Update successful
+        '400':
+          description: Bad request
+        '403':
+          description: Update not allowed
+        '404':
+          description: Unknown record
+        '500':
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+  /v1/credentialProfile:
+    get:
+      tags:
+        - Credential Profile
+      summary: Retrieve a list of credential profiles that match the passed in search criteria
+      operationId: findCredentialProfiles
+      parameters:
+        - name: expressions
+          in: query
+          description: The expressions to evaluate. Each expression
+            is described with the attribute's name, the operator
+            (one of ``<``, ``>``, ``=``, ``>=``, ``<=``, ``!=``)
+            and the attribute value.
+          required: true
+          schema:
+            $ref: '#/components/schemas/Expressions'
+        - name: attributes
+          in: query
+          description: The (optional) set of required attributes to retrieve
+          required: false
+          schema:
+            type: array
+            items:
+              type: string
+        - name: transactionId
+          in: query
+          description: The id of the transaction
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Read successful
+          $ref: '#/components/responses/CredentialProfilesResponse'
+        '400':
+          description: Bad request
+        '403':
+          description: Read not allowed
+        '404':
+          description: Unknown record
+        '500':
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+
+components:
+  responses:
+    CredentialRequestResponse:
+      description: The response for a credential request
+      content:
+        application/json:
+          schema:
+            allOf:
+              - $ref: '#/components/schemas/CredentialRequestId'
+              - $ref: '#/components/schemas/PersonId'
+              - $ref: '#/components/schemas/CredentialRequest'      
```

Comment:
I'm not very used to allOf pattern. What we have defined in PR & ABIS is nested entities. It avoids name collision and makes also a json more readable with data organized in topics, but this is only my opinion.
(Same comment applies below to other response types and to the CredentialRequest and Credential types.